### PR TITLE
Add hyperapp

### PR DESCRIPTION
**Pull request by zaceno**
By making a PR you are agreeing to that contributing a hello world project allows the repo/project owner to use the hello world app in examples for their own use.

- [x] only the most simple basic hello world app
- [x] remove all tests and commands for tests and dependencies for tests
- [x] must say "Hello World" on the main component
- [x] only include the minimal components (1 ideally)
- [x] no routing, not extras whatsoever
- [x] no styling
- [x] nothing else !
- [x] must work by running `npm install` and `npm start` (or equivalent)
  - no build necessary. library imported via es-modules from CDN (kinda like jqyery)
- [x] must be in the folder named `<tech-name>`
- [x] add yourself using the all-contributors bot `@allcontributors please add @<your-github-account> for code`
- [x] add build steps and which folder to serve in the README.md, Serve the `public` folder
  - It is all done and ready to serve in the root folder. Like the javascript and html examples.

### CODE REVIEW

File: hyperapp/hello-world.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import {h, app} from 'https://unpkg.com/hyperapp@2.0.4'
+
+app({
+    view: () => h('body', {}, 'hello world'),
```
Comment (jorgebucaran) [inline hyperapp/hello-world.js]: @zaceno If this is about minimalism, then I'd just `view: () => "hello world"` and not even import `h`.

Comment (zaceno) [reply hyperapp/hello-world.js]: Good point. Except then I’d have to also add a `<div id=“app”></div>` to the body and use `document.getElementById(‘app’)` for the node. I suppose that would be the preferred way to make a real app, but this is, in total, more minimal I think.

Comment (jorgebucaran) [reply hyperapp/hello-world.js]: Ah, missed `document.body`, in that case, could this be more minimal? 😄 

```js
import { app } from "https://unpkg.com/hyperapp@2.0.4"

app({ node, view: () => "hello world" })
```

```html
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <title>Hello World</title>
    <script src="./hello-world.js" type="module"></script>
  </head>
  <body><x id="node"></x></body>
</html>
```

Comment (jorgebucaran) [reply hyperapp/hello-world.js]: Or we could just go all the way...

```html
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <title>Hello World</title>
    <script type="module">
      import { app } from "https://unpkg.com/hyperapp@2.0.4"

      app({ node, view: () => "hello world" })
    </script>
  </head>
  <body>
    <x id="node"></x>
  </body>
</html>
```

Comment (zaceno) [reply hyperapp/hello-world.js]: That doesn’t work though. Does it?! How is node globally available?

Comment (jorgebucaran) [reply hyperapp/hello-world.js]: [Ah, but it does!](https://html.spec.whatwg.org/multipage/window-object.html#named-access-on-the-window-object) :trollface:

Comment (zaceno) [reply hyperapp/hello-world.js]: 🤯 all these years on the web and I didn’t know!

Comment (zaceno) [reply hyperapp/hello-world.js]: Cool. Ok Yeah it makes sense for hello world to just render hello world without rendering the body tags. Updated for that. Didn't use the fancy node-hack, cool as it is ;)

**Comment by zaceno** [comment]
@allcontributors please add @zaceno for code

**Review by johnpapa — APPROVED** [review]

**Pull request merged**